h/static/scripts/directive/excerpt.coffee

```diff
@@ -0,0 +1,35 @@
+###*
+# @ngdoc directive
+# @name excerpt
+# @restrict C
```

Comment:
I'm not a fan of class-invoked Angular magic. Preferably this would be a attribute-invoked or even element-invoked directive (see later comments for how that might work), even if that means doubling up on class names.